sljit_src/sljitUtils.c

```diff
@@ -208,6 +205,41 @@ static SLJIT_INLINE sljit_s32 open_dev_zero(void)
 
 #endif /* SLJIT_EXECUTABLE_ALLOCATOR || SLJIT_UTIL_GLOBAL_LOCK */
 
+#if (defined SLJIT_UTIL_STACK && SLJIT_UTIL_STACK) \
+	|| (defined SLJIT_EXECUTABLE_ALLOCATOR && SLJIT_EXECUTABLE_ALLOCATOR)
+
+#ifdef _WIN32
+
+SLJIT_INLINE static sljit_sw get_page_alignment(void) {
+	SYSTEM_INFO si;
+	static sljit_sw sljit_page_align;
```

Comment:
When this function is inlined multiple times, is it true that only a single shared static variable is created?

Reply:
since there is only compilation unit (with all other code included into sljitLir.c) it should be fine, but in case someone compiles sljitUtil independently it is safer if not inlined (it didn't help much anyway)